pkg/controller/volume/persistentvolume/scheduler_binder.go

```diff
@@ -160,26 +160,26 @@ func (b *volumeBinder) FindPodVolumes(pod *v1.Pod, node *v1.Node) (unboundVolume
 	}()
 
 	var (
-		matchedClaims     []*bindingInfo
```

Comment:
I prefer matchedClaims because it's more obvious that FindPodVolumes found matching PVs for them.

Reply:
Yeah, however matchedClaim may be misleading, it may cause users consider it as a pvc

Reply:
How about naming it "matchedBindings" then?

Reply:
Done! seems better!